test/e2e/framework/framework.go

```diff
@@ -272,3 +276,64 @@ func (f *Framework) AfterEach() {
 	}
 	f.OperatorInstalled = false
 }
+
+func printMySQLPodContainerLogs(f *Framework) {
+	pods, err := f.ClientSet.CoreV1().Pods(f.Namespace.Name).List(metav1.ListOptions{})
+	if err != nil {
+		Logf("Error retrieving pod list in namespace %s: %+v", f.Namespace.Name, err)
```

Comment:
Shouldn't we bomb out here, return an error or something instead of logging and continuing